.circleci/config.yml

```diff
@@ -67,7 +67,7 @@ jobs:
             && cmake .. -DCMAKE_BUILD_TYPE=Debug -DENABLE_COVERAGE=On -DCPACK_GENERATOR=DEB \
                 -DENABLE_COMPILER_WARNINGS=On -DENABLE_WERROR=Off -DCMAKE_EXPORT_COMPILE_COMMANDS=On \
                 -DCMAKE_CXX_FLAGS="-fuse-ld=lld" \
-                -DLOGGING_LEVEL=NONE \
+                -DLOGGING_LEVEL=INFO \
```

Comment:
so a while ago we disabled logging in the debug build because by default it logs way too much. then when we moved the guts from valhalla_convert_transit into the library, the file we added it to was part of -werror and so variables that are only used for logs become errors (since the logs are ifdef'd out from the compilers perspective). we could actually avoid making this change and use the preprocessor to conditionally make those variables over in the source file. @nilsnolde what do you think about that? should we try to keep logging at NONE? i kind of like logging at INFO in case its useful in the build

Reply:
> for some reason unused variables are treated as errors even if its not part of sources_with_warnings

was it using the ccache files then for years I guess?

> avoid making this change and use the preprocessor

yep, if the impact of INFO logging is so huge on debug builds then let's do that. hope it's not too many other long untouched files needing that, but we'll see:)

can you add that to the PR @chris-jpark ?

Reply:
i think the impact of INFO isnt huge. and i just looked and release is using TRACE which is crazy (but makes sure to test all code paths). unless you are totally opposed i would keep INFO